oedatamodel/latest/v112/datapackage/OEDataModel-concrete-datapackage/OEDataModel-concrete-datapackage.json

```diff
@@ -17,22 +17,16 @@
         "fundingAgencyLogo": "",
         "publisherLogo": ""
     },
-    "spatial": {
-        "location": "",
-        "extent": "",
-        "resolution": ""
-    },
-    "temporal": {
-        "referenceDate": "",
-        "timeseries": {
-            "start": "",
-            "end": "",
-            "resolution": "",
-            "alignment": "",
-            "aggregationType": ""
-        }
-    },
-    "sources": [
+  "spatial": {
+    "location": "",
+    "extent": "",
+    "resolution": ""
```

Comment:
Empty keys should be `null` or `todo` in oem151 and not `""`, see line 188-189 in the comments section.

Reply:
Thanks for starting to implement the requirements! 
Additionally, in oem151, for each dict in `resources/schema/fields`, the keys `isAbout` and `valueReference` exist, but this might be OK to leave out if an ontological annotation is not intended.

Reply:
I would argue to add them since they are part of the oemetadata v1.5.1.

Reply:
should be fixed with latest commit. thx